ui/src/app/pages/data/cohort/cohort-page.tsx

```diff
@@ -170,7 +170,7 @@ export const CohortPage = fp.flow(
       const {
         workspace: { id, namespace },
       } = this.props;
-      const cid = parseQueryParams(this.props.location.search).get('cohortId');
+      const cid = parseQueryParams().get('cohortId');
```

Comment:
confirmed that the `cohortId` queryParam continues to select the cohort, as expected